### ISSUE TITLE
[Bug] the learnable parameter beta seems to be missing in the ChanLayerNorm class of soundstorm.py line248

### ISSUE THREAD

**Issue by dbkest** [lucidrains/soundstorm-pytorch#30]
Hello, I seem to have found a bug. The ChanLayerNorm class in soundstorm.py line248 is designed to implement batchnorm1d? However, the learnable parameter beta seems to be missing in the actual code.

**Comment by dbkest** [comment]
thanks for your reply, i'm sorry to bother you.
